templates/Dockerfile-deb.tmpl

```diff
@@ -57,22 +57,25 @@ RUN echo 'deb http://apt.postgresql.org/pub/repos/apt/ %%release%%-pgdg main' >
         libxslt-dev \
         lintian \
         postgresql-server-dev-all \
-        wget \
+        # Add 12 package explicitly as *-all does not yet depend on 12 yet
+        postgresql-server-dev-12 \
+                wget \
     && rm -rf /var/lib/apt/lists/*
 
 # install jq to process JSON API responses
 RUN curl -sL https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64 \
          -o /usr/bin/jq \
     && chmod +x /usr/bin/jq
 
-# install latest Citus release to get header files
```

Comment:
This _entire_ `RUN` invocation should be removed. The header files were only needed in our build images for the old rebalancer package, and if we've retired that in favor of putting the functionality directly in Enterprise, there's no need for this command at all. Kill it (it takes a _long_ time, too, and brings in a lot of packages. Good riddance).

Reply:
I am not so sure if `pg_auto_failover_enterprise` needs any citus packages. I am unsure, so I want to keep this here for now.

Updating the comments to include this reason in the file